pycrostates/datasets/lemon/lemon.py

```diff
@@ -2,15 +2,14 @@
 
 import os
 
-import mne
 import numpy as np
 import pkg_resources as pkr
 import pooch
-from mne.io import BaseRaw
+from mne import create_info
+from mne.io import BaseRaw, RawArray
 
 from ...utils._checks import _check_type, _check_value
 from ...utils._config import get_config
-from ...utils._imports import import_optional_dependency
 
 
 def load_data(subject_id: str, condition: str):
```

Comment:
Since it returns a path (as a string), I think it would be better to match MNE API by naming this function `data_path` and returning a `pathlib.Path` instance.